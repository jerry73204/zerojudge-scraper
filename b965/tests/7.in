9 1 9
4
0
3
5
6
7
5
0
1
0 0 0 0 0 0 0 1 0