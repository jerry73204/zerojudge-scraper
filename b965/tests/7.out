9 1
4
0
3
5
6
7
5
0
1